<launch>

<param name="yaw_compensator/kp" value="4.0"/>
<param name="yaw_compensator/ki" value="0.4"/>
<param name="yaw_compensator/kd" value="3.5"/>
<param name="rotation_scale" value="2"/>

<node pkg="kingfisher_node" type="auto_kingfisher_node" name="auto_kingfisher_node" output="screen"/>

</launch>
